beacon-chain/db/kv/kv.go

```diff
@@ -40,8 +39,6 @@ type Store struct {
 	databasePath        string
 	blockCache          *ristretto.Cache
 	validatorIndexCache *ristretto.Cache
-	stateSlotBitLock    sync.Mutex
```

Comment:
@terencechain do we still need this ?

Reply:
nope, deprecated after #6461 

thanks for checking